BookShelfBL/ILibrary.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BookShelfBL
+{
+    /// <summary>
+    /// Contains a prototypes of all CRUD methods to be implemented in the business logic
+    /// </summary>
+    public interface ILibrary
```

Comment:
please for level4 split this big interface on small ones according to conversation that we had during the class